HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1673,7 +1685,7 @@ def set_unavailable_periods(unavailable_periods)
           if schedule_name == ColumnWaterHeaterSetpoint
             # Temperature of tank < 2C indicates of possibility of freeze.
             @tmp_schedules[schedule_name][i] = UnitConversions.convert(2.0, 'C', 'F') if @tmp_schedules[column_name][i] == 1.0
-          else
+          elsif ![SchedulesFile::ColumnHeatingSetpoint, SchedulesFile::ColumnCoolingSetpoint].include?(schedule_name)
```

Comment:
We don't want to zero out heating/cooling setpoints (even though they get effectively ignored since outage is enforced using availability schedules).